README.md

```diff
@@ -115,10 +88,10 @@ These are the arguments you can use to customize server behavior with default va
 | WORLD_MODE | 0 | Sets the world mode for the world. Can be Normal (0), Hard (1), Creative (2), Casual (4). |
 | SEASON | No Default | Overrides current season by setting to any of None (0), Easter (1), Halloween (2), Christmas (3), Valentine (4), Anniversary (5), CherryBlossom (6), LunarNewYear(7).<br/>**Do not set this env var if you want real date season.** |
 | GAME_ID | "" |  Game ID to use for the server. Need to be at least 28 characters and alphanumeric, excluding Y,y,x,0,O. Empty or not valid means a new ID will be generated at start. |
-| MAX_PLAYERS | 10 | Maximum number of players that will be allowed to connect to server. |
 | DATA_PATH | "/home/steam/core-keeper-data" | Save file location. |
+| MAX_PLAYERS | 10 | Maximum number of players that will be allowed to connect to server. |
 | SERVER_IP | No Default | Only used if port is set. Sets the address that the server will bind to. |
-| SERVER_PORT | No Default | What port to bind to. If not set, then the server will use the Steam relay network. If set the clients will connect to the server directly and the port needs to be open. |
+| SERVER_PORT | 27015 | What port to bind to. 27015 is the Steam relay port. |
```

Comment:
This is wrong, this port is the Game port, and whatever this is set to +1 is the Server Query port.
The steam relay port is given by the Steam Datagram Relay (SDR) when the server requests it.

Reply:
This can be changed to `no default` and modify the exposed docker port to the range of the SDR.